src/components/pacbio/PacbioPoolAliquotEdit.vue

```diff
@@ -39,10 +43,15 @@
         :error="errorsFor('volume')"
         :with-icon="isValidationExists('volume')"
       >
-        <traction-input v-model="volume" data-attribute="volume" placeholder="Volume" />
-        <div class="flex items-center px-1">
+        <traction-input
+          v-model="volume"
+          data-attribute="volume"
+          placeholder="Volume"
+          classes="min-w-128"
```

Comment:
Is this min-width needed?

Reply:
Sorry, that was part of a trial, will remove it